src/habluetooth/scanner.py

```diff
@@ -38,8 +38,13 @@
 if IS_LINUX:
     from bleak.backends.bluezdbus.advertisement_monitor import (
         AdvertisementMonitor,
-        OrPattern,
     )
+
+    try:
+        # bleak>1.0
+        from bleak.args.bluez import OrPattern
+    except ImportError:
+        from bleak.backends.bluezdbus.advertisement_monitor import OrPattern
     from bleak.backends.bluezdbus.scanner import BlueZScannerArgs
```

Comment:
`BlueZScannerArgs` should be handled the same as `OrPatterns`.

With Bleak v1.0.1, importing it from `bleak.backends.bluezdbus.scanner` will trigger a `DeprecationWarning`.